pkg/daemon/daemon.go

```diff
@@ -67,14 +67,14 @@ func (d *Daemon) SharedAbsMountPoint() string {
 	return filepath.Join(*d.RootMountPoint, d.SharedMountPoint())
 }
 
+// Mountpoint of Erofs/RafsV6 and per-image nydusd/rafs
```

Comment:
Does RAFS v6 support shared mode?

Reply:
fixed. New comment is 
// Mountpoint of per-image nydusd/rafs. It is a kernel mountpoint for each
// nydus meta layer. Each meta layer is associated with a nydusd.